examples/python/ocr/ocr.py

```diff
@@ -24,6 +27,10 @@
 
 SAMPLE_IMAGE_URLs = ["https://storage.googleapis.com/rerun-example-datasets/ocr/paper.png"]
 
+LayoutStructure: TypeAlias = tuple[
+    list[str], list[str], list[rrb.Spatial2DView], list[rrb.Spatial2DView], list[rrb.Spatial2DView]
+]
```

Comment:
A good rule of thumb is to only use tuples when it is immediate obvious what the different parts of it represent. I suggest using something a `namedtuple` or similar here instead

Reply:
…though I see now that the complicated tuple was already there since before so 🤷 
Only do the cleanup if you feel like it